orchestrator/test_runner/Cargo.toml

```diff
@@ -1,41 +0,0 @@
-[package]
```

Comment:
I believe there's also a `go.mod` for the test runner that we can drop as well.

Reply:
The testnet directory in the root of the repo? Should I remove the whole thing?